Neos.TimeableNodeVisibility/Classes/Domain/HandlingResult.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Neos\TimeableNodeVisibility\Domain;
+
+use Neos\ContentRepository\Core\Projection\ContentGraph\Node;
+
+class HandlingResult
+{
+    const RESULT_ENABLED = 'ENABLED';
+    const RESULT_DISABLED = 'DISABLED';
+
+    private function __construct(
+        public readonly Node $node,
+        public readonly string $result,
+    )
+    {
+    }
```

Comment:
sorry 😅 

For now we agreed to use this pattern FYI ^^ https://github.com/neos/neos-development-collection/pull/4605#discussion_r1358113177

```suggestion
    ) {
    }
```

Reply:
:+1: IDK why this pattern is configured in my auto-formattig rules... 🤷‍♂️